src/coreclr/jit/vartype.h

```diff
@@ -19,13 +19,7 @@ enum var_types_classification
     VTF_S   = 0x0040, // is a struct type
 };
 
-enum var_types : BYTE
-{
-#define DEF_TP(tn, nm, jitType, verType, sz, sze, asze, st, al, tf, howUsed) TYP_##tn,
-#include "typelist.h"
-#undef DEF_TP
-    TYP_COUNT
-};
+#include "vartypesdef.h"
```

Comment:
Yes, unfortunately. `vartype.h` has `varTypeIsValidHfaType` in it, which depends on `GlobalJitOptions`, which is defined in `jit.h`, which itself `#include`s `utils.h`, so I could not just `#include` `vartype.h` in `utils.h`. It would be great to not have to split `vartype.h` though. Maybe move `GlobalJitOptions` to its own header - but it is a tiny class?